static/src/js/pages/OrderOptionPage/OrderOptionPage.jsx

```diff
@@ -112,22 +133,20 @@ const OrderOptionPageHeader = () => {
         }
         pageType="secondary"
         primaryActions={
-          [
-            {
-              icon: FaEdit,
-              to: 'edit',
-              title: 'Edit',
-              iconTitle: 'A edit icon',
-              variant: 'primary'
-            },
-            {
-              icon: FaTrash,
-              onClick: () => toggleShowDeleteModal(true),
-              title: 'Delete',
-              iconTitle: 'A trash can icon',
-              variant: 'danger'
-            }
-          ]
+          [{
```

Comment:
There is no new-line here after the array open. But there is above. I prefer it with the newline but don't have a strong preference. Just noticed it was inconsistent.